### DISCUSSION TITLE
DeDrm installed, ADE keyfile not automatically retrieved

### DISCUSSION THREAD

**Discussion by mannaras** [noDRM/DeDRM_tools#648] [Q&A] [answered]
After a reinstall of Windows, I installed ADE 2.0 and Calibre 7.19.0, and the plugin DeDRM 10.0.9. I then added an ebook (ADE epub) to remove DRM. I wanted to let it automatically retrieve the key from ADE but this did not happen. It also didn't work via the customization dialogue. Can someone help?

**Comment by ElleKayEm** [comment]
Was ADE authorized with your Adobe ID?

**Reply by mannaras** [reply]
Yes.

**Reply by ElleKayEm** [reply]
Let's see a debug log of adding the book.

**Reply by mannaras** [reply]
[debug log.txt](https://github.com/user-attachments/files/17307541/debug.log.txt)

I removed the book and readded it for the debug log.

**Reply by ElleKayEm** [reply]
You downloaded the book with the current install of ADE, not DeACSM?

**Reply by mannaras** [reply]
Yes.

**Reply by ElleKayEm** [reply, marked as answer]
Not sure why it isn't working.  Maybe try deauthorizing and reauthorizing ADE:  https://helpx.adobe.com/digital-editions/kb/activation-errors-adobe-digital-editions.html

Scroll down for the Windows section about deleting the registry key.  Careful editing the registry.

**Comment by mannaras** [comment]
It worked - had to reainstall ADE completely, but have now correct file. Thank you very much!

**Reply by ElleKayEm** [reply]
Glad that worked!  Thanks for reporting back.
